metaflow/plugins/pypi/conda_environment.py

```diff
@@ -421,15 +421,16 @@ def bootstrap_commands(self, step_name, datastore_type):
         # Bootstrap conda and execution environment for step
         step = next(step for step in self.flow if step.name == step_name)
         id_ = self.get_environment(step).get("id_")
+        bootstrap_module_name = "fast_bootstrap" if CONDA_USE_FAST_INIT else "bootstrap"
         if id_:
             return [
                 "echo 'Bootstrapping virtual environment...'",
                 # We have to prevent the tracing module from loading,
                 # as the bootstrapping process uses the internal S3 client which would fail to import tracing
                 # due to the required dependencies being bundled into the conda environment,
                 # which is yet to be initialized at this point.
-                'DISABLE_TRACING=True python -m metaflow.plugins.pypi.bootstrap "%s" %s "%s" linux-64'
-                % (self.flow.name, id_, self.datastore_type),
+                'DISABLE_TRACING=True python -m metaflow.plugins.pypi.%s "%s" %s "%s" linux-64'
+                % (bootstrap_module_name, self.flow.name, id_, self.datastore_type),
```

Comment:
nit - can we maintain the singular - `metaflow.plugins.pypi.bootstrap` module and change the bootstrapping logic within it? easier to benchmark changes that way.

Reply:
done.